src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -825,22 +825,36 @@ public FeedbackSessionResultsBundle getFeedbackSessionResultsForStudent(
 
     public String getFeedbackSessionResultsSummaryAsCsv(
             String feedbackSessionName, String courseId,
-            String userEmail, String filterText, boolean isMissingResponsesShown, boolean isStatsShown)
+            String userEmail, String filterText, boolean isMissingResponsesShown,
+            boolean isStatsShown, String questionId)
             throws EntityDoesNotExistException, ExceedingRangeException {
 
         return getFeedbackSessionResultsSummaryInSectionAsCsv(
-                feedbackSessionName, courseId, userEmail, null, filterText, isMissingResponsesShown, isStatsShown);
+                feedbackSessionName, courseId, userEmail, null, filterText,
+                isMissingResponsesShown, isStatsShown, questionId);
     }
 
     public String getFeedbackSessionResultsSummaryInSectionAsCsv(
             String feedbackSessionName, String courseId, String userEmail,
-            String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown)
+            String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown, String questionId)
```

Comment:
Instead of adding the new parameter to the end, add it to the most appropriate place.